### DISCUSSION TITLE
When will next version will be released?

### DISCUSSION THREAD

**Discussion by lalitpagaria** [microsoft/presidio#790] [Q&A] [answered]
Just checking about the plan for the next version release.

**Comment by omri374** [comment, marked as answer]
Hi @lalitpagaria, we're planning a release in the next two weeks, together with the existing PRs we currently have.

**Reply by lalitpagaria** [reply]
Thanks @omri374

**Reply by omri374** [reply]
Sure, apologies for the delay. I know you are waiting for the update on the dependency version.

**Reply by lalitpagaria** [reply]
No problem.
I am getting dependence alert for NLTK lib but can't upgrade as they need >= 2021-x-x version.

**Reply by omri374** [reply]
@lalitpagaria version 2.2.23 was released.

**Reply by lalitpagaria** [reply]
Thanks @omri374
